app/move/controllers/create/move-details.js

```diff
@@ -37,12 +37,35 @@ class MoveDetailsController extends CreateBaseController {
         date: filters.formatDateWithDay(res.locals.TOMORROW),
       })
 
+      const itemForPrisonRecall = find(
+        req.form.options.fields.move_type.items,
+        { value: 'prison_recall' }
+      )
+      if (req.session.user.role === 'Prison') {
```

Comment:
We should avoid doing any login within this journey based on user role. It opens us up to problems if users have access to different location types.

Reply:
Good point, I'll amend accordingly.